src/map/chrif.c

```diff
@@ -298,9 +298,14 @@ int chrif_save(struct map_session_data *sd, int flag) {
 
 	chrif_bsdata_save(sd, (flag && (flag != 3)));
 
+	if (sd->state.storage_flag == 1)
+        intif_storage_save(sd,TABLE_STORAGE);
```

Comment:
Again, I think you merged incorrectly. Check your intif.h to make sure you defined the function in there.